gradle.properties

```diff
@@ -7,6 +7,7 @@ kotlin.code.style=official
 #Android
 android.useAndroidX=true
 android.nonTransitiveRClass=true
+android.defaults.buildfeatures.buildconfig=true
```

Comment:
FYI: this is required to have `BuildConfig` in MyApplication https://developer.android.com/build/releases/past-releases/agp-8-0-0-release-notes#default-changes